xbmc/epg/EpgInfoTag.cpp

```diff
@@ -749,3 +756,13 @@ void CEpgInfoTag::SetEpg(CEpg *epg)
 {
   m_epg = epg;
 }
+
+bool CEpgInfoTag::IsSeries(void) const
+{
+  return m_iFlags & EPG_TAG_FLAG_IS_SERIES > 0;
+}
```

Comment:
Yes (but do not copy the line you posted in your last comment - there is a const missing and the <code>inline</code> in front of the method definition is not needed.).

=> <code>bool IsSeries() const { return m_iFlags & (EPG_TAG_FLAG_IS_SERIES > 0); }</code>